### ISSUE TITLE
Archive the project?

### ISSUE THREAD

**Issue by Mic92** [nix-community/rnix-lsp#112]
It seems that there is no significant activity in this repository and https://github.com/oxalica/nil and https://github.com/nix-community/nixd replaced it for most users.
Should we just archive the repository?

**Comment by nyabinary** [comment]
Moving oxalica nil to nix-community seems like a good move first though if oxalica wants it to happen of course.

**Comment by IogaMaster** [comment]
> Moving oxalica nil to nix-community seems like a good move first though if oxalica wants it to happen of course.

Can agree!

**Comment by Mic92** [comment]
I don't think we need to wait for oxalica to move their project. They might prefer to keep things as they are and that's ok as long as the project is healthy. There is also nixd, which is already in this organization.
